### ISSUE TITLE
Fatally Missing Blocks

### ISSUE THREAD

**Issue by OrdinatorStouff** [Landmaster/PlusTiC#85]
We're having a weird issue on the ATC server. Mods, Configs, Scripts, all exactly the same on client and server. Sometimes users can connect with no issues. Other times, users will get Fatally Missing Blocks

For example, I did a fresh copy+paste of exact folders on server and was able to connect. I got disconnected for an unrelated mek crash and when I re-launched (nothing changed) got the fatally missing blocks.

Some of the devs in the Discord think that it could have something to do with load order being different on server (linux) than on clients (windows) but nothing is certain

PlusTIC 6.0.0.4
Tinkers 1.12-2.7.2.15
Forge 2443

We **don't** have moar tinkers or any other tinker addons that I'm aware of but just in case here's the modlist anyway: https://pastebin.com/78fse9Z8

```
[07:10:39] [Netty Client IO #1/INFO] [FML]: Registry IRecipe: Found a missing id from the world tconstruct:tools/forge/blockosmiridium
[07:10:39] [Netty Client IO #1/INFO] [FML]: Registry IRecipe: Found a missing id from the world plustic:metals/osmiridium_ingot_nugget
[07:10:39] [Netty Client IO #1/INFO] [FML]: Registry IRecipe: Found a missing id from the world plustic:metals/osmiridium_ingot
[07:10:39] [Netty Client IO #1/INFO] [FML]: Registry IRecipe: Found a missing id from the world plustic:metals/osmiridium_block
[07:10:39] [Netty Client IO #1/INFO] [FML]: Registry IRecipe: Found a missing id from the world plustic:metals/osmiridium_nugget
[07:10:39] [Netty Client IO #1/INFO] [FML]: Registry Item: Found a missing id from the world plustic:plustic.molten_osmiridium
[07:10:39] [Netty Client IO #1/INFO] [FML]: Registry Item: Found a missing id from the world plustic:osmiridiumnugget
[07:10:39] [Netty Client IO #1/INFO] [FML]: Registry Item: Found a missing id from the world plustic:osmiridiumingot
[07:10:39] [Netty Client IO #1/INFO] [FML]: Registry Item: Found a missing id from the world plustic:osmiridiumblock
[07:10:39] [Netty Client IO #1/ERROR] [FML]: Unidentified mapping from registry minecraft:blocks
[07:10:39] [Netty Client IO #1/ERROR] [FML]:     plustic:plustic.molten_osmiridium: 3531
[07:10:39] [Netty Client IO #1/ERROR] [FML]:     plustic:osmiridiumblock: 3657
[07:10:39] [Netty Client IO #1/ERROR] [FML]: Unidentified mapping from registry minecraft:recipes
[07:10:39] [Netty Client IO #1/ERROR] [FML]:     plustic:metals/osmiridium_ingot_nugget: 6703
[07:10:39] [Netty Client IO #1/ERROR] [FML]:     plustic:metals/osmiridium_nugget: 6702
[07:10:39] [Netty Client IO #1/ERROR] [FML]:     plustic:metals/osmiridium_ingot: 6704
[07:10:39] [Netty Client IO #1/ERROR] [FML]:     tconstruct:tools/forge/blockosmiridium: 7271
[07:10:39] [Netty Client IO #1/ERROR] [FML]:     plustic:metals/osmiridium_block: 6705
[07:10:39] [Netty Client IO #1/ERROR] [FML]: Unidentified mapping from registry minecraft:items
[07:10:39] [Netty Client IO #1/ERROR] [FML]:     plustic:osmiridiumnugget: 9848
[07:10:39] [Netty Client IO #1/ERROR] [FML]:     plustic:osmiridiumblock: 7825
[07:10:39] [Netty Client IO #1/ERROR] [FML]:     plustic:plustic.molten_osmiridium: 4641
[07:10:39] [Netty Client IO #1/ERROR] [FML]:     plustic:osmiridiumingot: 10758
[07:10:39] [Netty Client IO #1/ERROR] [FML]: Network Disconnect: Fatally missing blocks and items
[07:10:40] [Netty Client IO #1/FATAL] [FML]: Failed to connect to server: there are 22 missing blocks and items
```

**Comment by Landmaster** [comment]
@OrdinatorStouff Can you post a full log?

**Comment by OrdinatorStouff** [comment]
I shut down twitch launcher from tray, deleted these folders from my profile then copied them from the server (that's online and some players can connect to) via FTP. 
```
config
mods
local
scripts
```
I then re-opened twitch and verified I'm using Forge 2443 for client version and launched. Could not connect to server w/ "Fatally Missing."

Requested full logs:
[fml-client-latest.log](http://paste.ubuntu.com/25236996/)
[fml-junk-earlystartup.log](http://paste.ubuntu.com/25237023/)

**Comment by Landmaster** [comment]
Fixed in 6.0.0.5
